src/pe_mailer/pe_message.py

```diff
@@ -0,0 +1,137 @@
+
+Note: The report is encrypted with your Cyber Hygiene password.
+
+Thank you,
+CISA Cyber Assessments - Posture and Exposure
+Cybersecurity and Infrastructure Security Agency
+vulnerability@cisa.dhs.gov
+
+WARNING: This document is FOR OFFICIAL USE ONLY (FOUO). It contains information that may be exempt from public release under the Freedom of Information Act (5 U.S.G. 552). It is to be controlled, stored, handled, transmitted, distributed, and disposed of in accordance with CISA policy relating to FOUO information and is not to be released to the public or other personnel who do not have a valid 'need-to-know' without prior approval of an authorized CISA official.
+"""
+
+    HtmlBody = """<html>
+<head></head>
+<body>
+<p>Greetings,</p>
+
+<p>The attached Posture and Exposure (P&E) report is the result of a
+CISA Cyber Assessments service that provides actionable information
+about public exposures and security posture weaknesses.</p>
+
+<p>All of the findings and information are derived from public
+information that is currently available. No scanning has occurred
+for this service.</p>
+
+<p>The report will initially be delivered twice per month, but it
+will be updated and enhanced to integrate more data sources and be
+sent with greater frequency in the future. The P&E report is for your
+situational awareness as a supplement to other threat reports you may
+have internally or externally. No action is required, but your feedback
+and questions are more than welcome.</p>
+
+<p>Note: The report is encrypted with your Cyber Hygiene password.</p>
+
+<p>Thank you,<br>
+CISA Cyber Assessments - Posture and Exposure<br>
+Cybersecurity and Infrastructure Security Agency<br>
+<a href="mailto:vulnerability@cisa.dhs.gov">vulnerability@cisa.dhs.gov</a></p>
+
+<p>WARNING: This document is FOR OFFICIAL USE ONLY (FOUO). It contains information that may be exempt from public release under the Freedom of Information Act (5 U.S.G. 552). It is to be controlled, stored, handled, transmitted, distributed, and disposed of in accordance with CISA policy relating to FOUO information and is not to be released to the public or other personnel who do not have a valid 'need-to-know' without prior approval of an authorized CISA official.</p>
+</body>
+</html>
+"""
+
+    def __init__(
+        self,
+        pdf_filename,
+        report_date,
+        to_addrs,
+        from_addr=Message.DefaultFrom,
+        cc_addrs=Message.DefaultCc,
+        bcc_addrs=Message.DefaultBcc,
+    ):
+        """Construct an instance.
+
+        Parameters
+        ----------
+        pdf_filename : str
+            The filename of the PDF file that is the Posture and
+            Exposure report corresponding to this message.
+
+        report_date : str
+            The date corresponding to the Posture and Exposure
+            report attachment. We have been using dates of the
+            form December 12, 2017.
```

Comment:
I'd suggest something slightly different than what @dav3r did:
```suggestion
            report attachment. Specify dates using the format "December 12, 2017".
```

Reply:
Also good - I'm not tied to my suggestion.